### PR TITLE
BaSP-TG-18: Created delete test for time sheets

### PR DESCRIPTION
**Pull request by disimoneleandro**

### CODE REVIEW

File: src/seed/employees.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import mongoose from 'mongoose';
+
+export default [{
+  _id: mongoose.Types.ObjectId('634c68f3658f142935ea7f6e'),
```
Comment (lmalvasia) [inline src/seed/employees.js:4]: We should have almost 5 elements here

File: src/seed/projects.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import mongoose from 'mongoose';
+
+export default [{
+  _id: mongoose.Types.ObjectId('6352dae0282ae0599b7acd92'),
```
Comment (lmalvasia) [inline src/seed/projects.js:4]: We should have almost 5 elements here

File: src/seed/tasks.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import mongoose from 'mongoose';
+
+export default [{
+  _id: mongoose.Types.ObjectId('6352dae0282ae0599b7acd91'),
```
Comment (lmalvasia) [inline src/seed/tasks.js:4]: We should have almost 5 elements here

File: src/test/time-sheets.test.js [hunk 4]
```diff
@@ -1,3 +1,36 @@
...
+    await Projects.collection.insertMany(projectsSeed);
+    await Tasks.collection.insertMany(tasksSeed);
+  });
+
+  describe('DELETE /time-sheets', () => {
+    test('should delete user when user send an valid id', async () => {
+      const response = await request(app).delete(`/time-sheets/${validId}`).send();
+      expect(response.status).toBe(200);
+      expect(response.body.error).toBeFalsy();
+    });
```
Comment (lmalvasia) [inline src/test/time-sheets.test.js:28]: Add a new line here to separate the tests

File: src/seed/projects.js (hunk 2 above)
Comment (disimoneleandro) [reply src/seed/projects.js:4]: five entities were added

File: src/seed/tasks.js (hunk 3 above)
Comment (disimoneleandro) [reply src/seed/tasks.js:4]: five entities were added

File: src/test/time-sheets.test.js (hunk 4 above)
Comment (disimoneleandro) [reply src/test/time-sheets.test.js:28]: added space between tests

File: src/seed/employees.js (hunk 1 above)
Comment (disimoneleandro) [reply src/seed/employees.js:4]: five elements were added

**Review by GinaSchP — APPROVED** [review]

**Review by lmalvasia — APPROVED** [review]

**Pull request merged**